Makefile

```diff
@@ -64,24 +64,20 @@ benchmark-python-local:
 	FEAST_USAGE=False IS_TEST=True FEAST_IS_LOCAL_TEST=True python -m pytest --integration --benchmark  --benchmark-autosave --benchmark-save-data sdk/python/tests
 
 test-python:
```

Comment:
consider renaming it to `test-python-unit` ?

Reply:
good idea. done